src/features/parameters/common/parameter-layout.tsx

```
@@ -0,0 +1,125 @@
+// Context for XS Screen
+interface ParameterLayoutContextValue {
+    isXsScreen: boolean;
+}
+
+const ParameterLayoutContext = createContext<ParameterLayoutContextValue>({ isXsScreen: false });
+
+export const useParameterLayoutContext = () => useContext(ParameterLayoutContext);
+
+export function ParameterLayoutProvider({ isXsScreen, children }: { isXsScreen: boolean; children: ReactNode }) {
+    return (
+        <ParameterLayoutContext.Provider value={{ isXsScreen }}>
```

ESLint

[error] The object passed as the value prop to the Context provider (at line 26) changes every render. To fix this consider wrapping it in a useMemo hook.

(react/jsx-no-constructed-context-values)